src/engraving/rendering/dev/tdraw.cpp

```diff
@@ -283,6 +284,8 @@ void TDraw::drawItem(const EngravingItem* item, draw::Painter* painter)
         break;
     case ElementType::STEM_SLASH:           draw(item_cast<const StemSlash*>(item), painter);
         break;
+    case ElementType::STICKING:             draw(item_cast<const Sticking*>(item), painter);
```

Comment:
Are we sure that we don't need it for `single`?

Reply:
added to single